### DISCUSSION TITLE
Problem discord notifs for separate notifications in each channel !

### DISCUSSION THREAD

**Discussion by Dilolcyy** [jef/streetmerchant#2402] [Help] [answered]
Hello everyone, anyone to help me for discord notifications for settings pls ?
it sends the notification on the webhook but not on roles separately
Where is the problem ?

![image](https://user-images.githubusercontent.com/82901639/115533273-e07cc700-a296-11eb-9d50-39ef94a7171a.png)

And my roles definid is here and i copied for every role the ID ...
![image](https://user-images.githubusercontent.com/82901639/115533316-ec688900-a296-11eb-9940-96a062b6aa94.png)

here is my discord announcements defined, I would like it to fall in for each role defined
![image](https://user-images.githubusercontent.com/82901639/115533384-f7bbb480-a296-11eb-9492-22b6f1cfb0cb.png)

![image](https://user-images.githubusercontent.com/82901639/115533862-757fc000-a297-11eb-8954-fb0e39a72c01.png)

I would like how this should work for separate items for different channels?

**Comment by Mr-Tech-13** [comment, marked as answer]
from my understanding you need a web hook for each channel, so a separate folder for each item you want to check. Or you have a single channel and it alerts the different roles as stuff comes in
<img width="209" alt="Screen Shot 2021-04-22 at 9 58 21 AM" src="https://user-images.githubusercontent.com/81822275/115727088-4af23d80-a351-11eb-9c2b-2cc655ffa9dc.png">

**Reply by Dilolcyy** [reply]
Yes that is exactly what I would like to do.  With each notification I want it to fall in the concerned channel with the defined role.  So I have to create separate folders for each?  And start streetmerchant separately for each folder?  It is not going to be too loaded in terms of the CPU usage of the PC

**Comment by Mr-Tech-13** [comment]
Yes you do, it will load a lot of the cpu

**Reply by Dilolcyy** [reply]
OK! Thank you.. If you have other solutions afterwards I'll be there :)
